src/components/ShareButtons/ShareButtons.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+import { Copy } from 'lucide-react';
+import { toast } from 'sonner';
+import { Button } from '../ui/button';
+import LinkedinIcon from '../icons/LinkedinIcon';
+import TwitterIcon from '../icons/TwitterIcon';
+
+interface ShareButtonsProps {
+  title: string;
+  url: string;
+}
+
+const ShareButtons: React.FC<ShareButtonsProps> = ({ title, url }) => {
```

Comment:
The `ShareButtons` component is well-implemented, but it could be more extensible. For example, you could create a configuration array for the share buttons, which would make it easier to add or remove social media platforms in the future. This would also make the code more maintainable.